pkg/kp/rcstore/store.go

```diff
@@ -82,3 +85,11 @@ func RCUpdateLockPath(rcId fields.ID) (string, error) {
 
 	return path.Join(kp.LOCK_TREE, rcPath, "update"), nil
 }
+
+func IsNotExist(err error) bool {
+	if err == NoReplicationController {
```

Comment:
nah you're right :) it used to be a type assertion on a type and i didn't clean up all the way